source/includes/document-filtering-intro.rst

```diff
@@ -0,0 +1,5 @@
+Starting in ``mongosync-beta`` 1.8, you can filter documents to migrate 
```

Comment:
Nit: I find this first sentence is a bit confusing, what do you think about something like:

```
Starting in ``mongosync-beta`` 1.8, you can selectively migrate documents based on specific conditions.
```

Reply:
I can adjust the copy to something similar to this! Might just make some tweaks to align with our docs style guide.